### PR TITLE
chore: stats script parameters

### PR DESCRIPTION
**Pull request by mcdurdin**
Someone asked for this last time I worked on stats. So a small sidetrack later, here 'tis.

@keymanapp-test-bot skip

### CODE REVIEW

File: resources/stats/stats.sh [hunk 1]
```diff
@@ -15,20 +15,47 @@ cd "$THIS_SCRIPT_PATH"
 
 platforms=(android/ ios/ linux/ mac/ web/ windows/ developer/ core/ common/models/)
 
-startdate=2022-05-01
-stable=stable-15.0
+builder_describe "Stats collector for Keyman PRs and issues" \
+  stats+ \
+  "--start-date,-d=startDate       Start date for collecting data, yyyy-mm-dd" \
+  "--start-sprint,-s=startSprint   Or, first sprint to start collecting data for" \
+  "--stable,-S=stableBranch        Branch name for previous stable release"
```
Comment (darcywong00) [inline resources/stats/stats.sh:22]: Is there a way to note --stable is a required parameter, along with either --start-date or --start-sprint?

Comment (mcdurdin) [reply resources/stats/stats.sh:22]: I don't think we do that in builder scripts at present. We could add it to the description line?

(I don't want to do any changes to the builder scripts here, and I think until we actually need required options for other build scripts it's not worth changing builder for this little utility script.)

File: resources/stats/stats.sh [hunk 2]
```diff
@@ -15,20 +15,47 @@ cd "$THIS_SCRIPT_PATH"
 
 platforms=(android/ ios/ linux/ mac/ web/ windows/ developer/ core/ common/models/)
 
-startdate=2022-05-01
-stable=stable-15.0
+builder_describe "Stats collector for Keyman PRs and issues" \
```
Comment (mcdurdin) [inline resources/stats/stats.sh:18]:
```suggestion
builder_describe "Stats collector for Keyman PRs and issues

* You must specify either --start-date (-d) or --start-sprint (-s).
" \
```

File: resources/stats/stats.sh (hunk 1 above)
Comment (mcdurdin) [reply resources/stats/stats.sh:22]:
```suggestion
  "--stable,-S=stableBranch        Branch name for previous stable release (required)"
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.158-alpha](https://keyman.com/downloads/releases/alpha/17.0.158)

**Pull request merged**